README.md

```diff
@@ -1,2 +1,68 @@
+
 # react-native-shopify
-React Native bridge to Shopify Buy SDK
+
+## Getting started
+
+`$ npm install react-native-shopify --save`
+
+### Including Mobile-BUY-SDK
+
+Include the Shopify Mobile Buy SDK in your project to make it available to the bridge.
+Follow the instructions on their Github page to get started. For example,
+the recommended and easiest way for iOS is to install it as a Pod. This project will look
+for headers in the Pods directory.
+
+### Mostly automatic installation
+
+`$ react-native link react-native-shopify`
+
+### Manual installation
+
+
+#### iOS
+
+1. In XCode, in the project navigator, right click `Libraries` ➜ `Add Files to [your project's name]`
+2. Go to `node_modules` ➜ `react-native-shopify` and add `RNShopify.xcodeproj`
+3. In XCode, in the project navigator, select your project. Add `libRNShopify.a` to your project's `Build Phases` ➜ `Link Binary With Libraries`
+4. Run your project (`Cmd+R`)<
+
+#### Android
+
+1. Open up `android/app/src/main/java/[...]/MainActivity.java`
+  - Add `import com.reactnativeshopify.RNShopifyPackage;` to the imports at the top of the file
+  - Add `new RNShopifyPackage()` to the list returned by the `getPackages()` method
+2. Append the following lines to `android/settings.gradle`:
+  	```
+  	include ':react-native-shopify'
+  	project(':react-native-shopify').projectDir = new File(rootProject.projectDir, 	'../node_modules/react-native-shopify/android')
+  	```
+3. Insert the following lines inside the dependencies block in `android/app/build.gradle`:
+  	```
+      compile project(':react-native-shopify')
+  	```
+
+#### Windows
```

Comment:
@SoHotSoup will be disappointed, but you are right: farewell Windows. It came with the creator tool anyway :).